client/state/sites/plans/reducer.js

```diff
@@ -1,39 +1,86 @@
 /**
+ * External dependencies
+ */
+import omit from 'lodash/object/omit';
+
+/**
  * Internal dependencies
  */
 import {
-	FETCH_SITE_PLANS,
-	FETCH_SITE_PLANS_COMPLETED,
-	REMOVE_SITE_PLANS
-} from './action-types';
-import omit from 'lodash/object/omit';
+	SITE_PLANS_FETCH,
+	SITE_PLANS_FETCH_COMPLETED,
+	SITE_PLANS_FETCH_FAILED,
+	SITE_PLANS_REMOVE,
+	SITE_PLANS_TRIAL_CANCEL,
+	SITE_PLANS_TRIAL_CANCEL_COMPLETED,
+	SITE_PLANS_TRIAL_CANCEL_FAILED
+} from 'state/action-types';
 
 export const initialSiteState = {
+	data: null,
 	error: null,
 	hasLoadedFromServer: false,
 	isFetching: false,
-	data: null
+	isUpdating: false
```

Comment:
The [`Site Redirect` store](https://github.com/Automattic/wp-calypso/blob/master/client/lib/domains/site-redirect/reducer.js) also uses this approach but I'll let you guess who coded it :).

I'll merge `isFetching` and `isUpdating` into `isRequesting`.